launcher/package/rpm/MultiMC5.spec

```diff
@@ -23,22 +29,29 @@ mkdir -p %{buildroot}/opt/multimc
 install -m 0644 ../ubuntu/multimc/opt/multimc/icon.svg %{buildroot}/opt/multimc/icon.svg
 install -m 0755 ../ubuntu/multimc/opt/multimc/run.sh %{buildroot}/opt/multimc/run.sh
 mkdir -p %{buildroot}/%{_datadir}/applications
-install -m 0644 ../ubuntu/multimc/usr/share/applications/multimc.desktop %{buildroot}/%{_datadir}/applications/multimc.desktop
+desktop-file-install --dir=%{buildroot}%{_datadir}/applications ../ubuntu/multimc/usr/share/applications/multimc.desktop
```

Comment:
usage of desktop-file-install is demanded to ensure .desktop safety and spec compliance    
https://docs.fedoraproject.org/en-US/packaging-guidelines/#_desktop_file_install_usage